### PR TITLE
Update german.md

### PR DESCRIPTION
**Pull request by dolibit-ut**
npm install create-nocodb-app

### CODE REVIEW

File: scripts/markdown/readme/languages/german.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ docker run -d --name nocodb -p 8080:8080 nocodb/nocodb:latest
 ### Verwenden von NPM
 
 ```
-npm create-nocodb-app
+npm install create-nocodb-app
```
Comment (wingkwong) [inline scripts/markdown/readme/languages/german.md:68]: It should be `npx create-nocodb-app`

Comment (dolibit-ut) [reply scripts/markdown/readme/languages/german.md:68]: i think it works both

Comment (dolibit-ut) [reply scripts/markdown/readme/languages/german.md:68]: have now also added NPX

Comment (wingkwong) [reply scripts/markdown/readme/languages/german.md:68]: Actually they are different. We want to guide users to create project in an interactive way. (See the first screenshot). Besides, it should follow the main documentation (EN version). 

Using `npx create-nocodb-app`

image

Using `npm install create-nocodb-app`

image

File: scripts/markdown/readme/languages/german.md [hunk 2]
```diff
@@ -62,12 +62,18 @@ docker run -d --name nocodb -p 8080:8080 nocodb/nocodb:latest
...
+### Verwenden von NPM 
 
 ```
 npm install create-nocodb-app
 ```
 
+### Verwenden von NPX 
+
+```
+npx install create-nocodb-app
```
Comment (wingkwong) [inline scripts/markdown/readme/languages/german.md:74]: You would get `npm ERR! could not determine executable to run`. Besides, please remove that. See the previous comment.

**Comment by o1lab** [comment]
@all-contributors please add @dolibit-ut  for code

File: scripts/markdown/readme/languages/german.md (hunk 2 above)
Comment (o1lab) [reply scripts/markdown/readme/languages/german.md:74]: Have corrected this after the merge.

**Pull request merged**